src/server_manager/web_app/ui_components/outline-region-picker-step.ts

```diff
@@ -123,7 +126,7 @@ export class OutlineRegionPicker extends LitElement {
               ${this.selectedLocationId === item.id ? html`<iron-icon icon="check-circle"></iron-icon>` : ''}
             </div>
             <img class="flag" src="${item.flag}">
-            <div class="city-name">${item.name}</div>
+            <div class="city-name">${this._formatLocation(item, this.language)}</div>
```

Comment:
Something like this, but taking care of places that are countries.

```suggestion
            <div class="city-name">${this.localize(this.name_msg)}<br>${this.countryName(this.language, item.country)}</div>
```

Reply:
I wish we could intercept localize to have special logic if the message was `country-$code`. Or have extra methods, like `this.localize.countryName(code)`. The polymer localize behavior is quite restrictive.

Reply:
Perhaps instead of setting localize functions we could set a i18n object in the DOM. Then you could do `i18n.language`, or `i18n.format()` . We could create that object in AppRoot.setLanguage() and pass around.

Or for backward compatibility the object could be callable to invoke the localize behavior.

Reply:
I reorganized this similarly to your suggestion.